### DISCUSSION TITLE
Assertion question

### DISCUSSION THREAD

**Discussion by busslina** [GregoryConrad/rearch-dart#129] [General]
I know that assertions are only executed on debug mode, but, why don't you make it fail on production? E.G. [This one](https://github.com/GregoryConrad/rearch-dart/discussions/128)

**Comment by GregoryConrad** [comment]
Because I didn't want to take a running application completely down on something that may be semi-recoverable. But regardless, any dev build, especially in Flutter, will have asserts throw and make it apparent something is wrong.
